### PR TITLE
Update the acceptance of SQE, and the numerical CQA paper NRN  from KDD 2023

### PR DESCRIPTION
**Pull request by marcos0318**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -143,9 +145,9 @@ This repo follows the Neural Query Engine taxonomy proposed in the paper (Figure
 </details>
 
 <details>
-  <summary>Discrete + Continuous (Entities + string/numerical Literals) (0)</summary>
+  <summary>Discrete + Continuous (Entities + numerical value) (1)</summary>
```
Comment (migalkin) [inline README.md]: I would rather not change the original classification in favor of one particular work. The big vision is still to be able to run queries against both strings and numerical literals

Comment (migalkin) [reply README.md]: And if we add NRN here, then LitCQD should go into this category first :)

Comment (marcos0318) [reply README.md]: I did not pay enough attention to the categories. I totally agree with you! Thank you for your reminder!

Comment (marcos0318) [reply README.md]: Is there anything that I can help with?

Comment (marcos0318) [reply README.md]: Should I add a new pull request? I am not very familiar with the pull request system

Comment (marcos0318) [reply README.md]: Oh, I am reading the github doc, and I think I am able to tackle it

**Comment by marcos0318** [comment]
Hi, Thank you for your review~~ I have updated it accordingly.

**Pull request merged**